internal/testplanet/uplink.go

```diff
@@ -166,8 +166,15 @@ func (uplink *Uplink) DialOverlay(destination Peer) (overlay.Client, error) {
 }
 
 // Upload data to specific satellite
-func (uplink *Uplink) Upload(ctx context.Context, satellite *satellite.Peer, bucket string, path storj.Path, data []byte) error {
+func (uplink *Uplink) Upload(ctx context.Context, satellite *satellite.Peer, redundancy *uplink.RSConfig, bucket string, path storj.Path, data []byte) error {
 	config := uplink.getConfig(satellite)
+	if redundancy != nil {
+		config.RS.MinThreshold = redundancy.MinThreshold
```

Comment:
This changes the existing behavior of the uplink, which is unexpected, given that it's an argument.